sonar-plugin-api/src/main/java/org/sonar/api/utils/Durations.java

```diff
@@ -36,6 +32,16 @@
 @ComputeEngineSide
 public class Durations {
 
+  private static final String MINUTES_FORMAT = "%smin";
+  private static final String HOURS_FORMAT = "%sh";
+  private static final String DAYS_FORMAT = "%sd";
+
+  private static final int HOURS_IN_DAY = 8;
+
+  /**
+   * @deprecated since 6.3, only one format is available
+   */
+  @Deprecated
   public enum DurationFormat {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)